internal/rootcoord/quota_center.go

```diff
@@ -1182,6 +1182,15 @@ func (q *QuotaCenter) calculateRates() error {
 func (q *QuotaCenter) resetAllCurrentRates() error {
 	clusterLimiter := newParamLimiterFunc(internalpb.RateScope_Cluster, allOps)()
 	q.rateLimiter = rlinternal.NewRateLimiterTree(clusterLimiter)
+
+	enablePartitionRateLimit := false
+	for rt := range getRateTypes(internalpb.RateScope_Partition, allOps) {
+		r := quota.GetQuotaValue(internalpb.RateScope_Partition, rt, Params)
+		if Limit(r) != Inf {
+			enablePartitionRateLimit = true
+		}
+	}
```

Comment:
It implies `Inf` are the same as `nil` for partition limiter. And this context need to be maintained by `rate_limiter_tree` instead of this function. 

Suggest to create a new function in `rate_limiter_tree` to skip partition limiter's construction.

Reply:
It's OK to ignore this comment, not a blocker

Reply:
Alright, we can complete it once the quota center is refactored